apps/api/v2/src/modules/apps/services/gcal.service.ts

```diff
@@ -4,9 +4,11 @@ import { google } from "googleapis";
 import { z } from "zod";
 
 @Injectable()
-export class GcalService {
+export class GCalService {
   private logger = new Logger("GcalService");
 
+  private gcalResponseSchema = z.object({ client_id: z.string(), client_secret: z.string() });
```

Comment:
Since you changed the class to GCalService with a capital G, want to do the same here?